skimage/feature/_canny.py

```diff
@@ -55,8 +55,12 @@ def _preprocess(image, mask, sigma, mode, cval):
     on the mask to recover the effect of smoothing from just the significant
     pixels.
     """
-    gaussian_kwargs = dict(sigma=sigma, mode=mode, cval=cval,
-                           preserve_range=False)
+    gaussian_kwargs = dict(
+        sigma=sigma,
+        mode=mode,
+        cval=cval,
+        preserve_range=False
```

Comment:
```suggestion
        preserve_range=False,
```

Reply:
This type of feedback is verboten :joy:

Please update the black config if you want such changes to be made.

Reply:
Oh, that's right, for some reason skimage can't agree to use black. Which is pretty dumb, because then we have to deal with this type of thing.

Reply:
Formally withdrawing this suggestion. :bow:

Reply:
Hehe, I'm teasing you, of course, but also part serious ;)